qiita_db/handlers/reference.py

```diff
@@ -0,0 +1,71 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from tornado.web import RequestHandler
+
+import qiita_db as qdb
+
+
+def _get_reference(r_id):
+    """Returns the reference with the given id if exists
+
+    Parameters
+    ----------
+    qiita_db.reference.Reference, bool, string
+        The requested reference or None
+        Whether if we could get the reference object or not
+        Error message in case we counldn't get the reference object
+    """
+    try:
+        reference = qdb.reference.Reference(r_id)
+    except qdb.exceptions.QiitaDBUnknownIDError:
+        return None, False, 'Reference does not exist'
+    except qdb.exceptions.QiitaDBError as e:
+        return None, False, 'Error instantiating the reference: %s' % str(e)
+
+    return reference, True, ''
+
+
+class ReferenceFilepathsHandler(RequestHandler):
+    def get(self, reference_id):
+        """Retrieves the filepath information of the given reference
+
+        Parameters
+        ----------
+        reference_id : str
+            The id of the reference whose filepath information is being
+            retrieved
+
+        Returns
+        -------
+        dict
+            Format:
+            {'success': bool,
+             'error': str,
+             'filepaths': list of (str, str)}
+            - success: whether the request is successful or not
```

Comment:
I think for consistency we should leave it here (this is how there are in the other handlers documentation).
Since there is an issue that @ElDeveloper opened to change the behavior of handlers (i.e. do not return the success bool, just return error codes) I suggest to leave as it is and fix when fixing that other issue.